app/keepers/keepers.go

```diff
@@ -373,7 +401,12 @@ func (ak *SecretAppKeepers) InitCustomKeepers(
 		ibcpacketforwardkeeper.DefaultRefundTransferPacketTimeoutTimestamp,
 	)
 	transferStack = ibcfee.NewIBCMiddleware(transferStack, ak.IbcFeeKeeper)
+	// todo: this is ugly since the IBCModule interface on the switch module is implemented with pointers, try without pointers instead
+	var stackWithSwitch ibcswitch.IBCModule
```

Comment:
fixed, it was an ugly solution because of the implemetation of the interface with `func (m *IBCModule) ...` (with a pointer) instead of without a pointer.